app/helpers/reactjs_helper.rb

```diff
@@ -1,8 +1,25 @@
 module ReactjsHelper
+  def react_component(name, opts = { data: [], flatten_data: false })
+    data = opts[:data]
+    data = data.to_json if data.is_a?(Hash)
+
+    attributes = {
+      name: name,
+      'data-props': data,
+    }
+
+    attributes[:flatten] = true if opts[:flatten_data]
+
+    content_tag('react-component', '', attributes).html_safe
+  end
+
   def mount_react_component(name, selector, data = [], opts = {})
```

Comment:
Just realized: what about the places where we use the `mount_react_component` on a different place than is the actual element. Now it is possible to put the mounter anywhere as it is js tag :thinking:

Reply:
Yes I thought about it and found out the `ReactApp` component suffer from this issue. (but somehow everything still functioning).

Anyway, let's keep those changes for benchmarking only